proto/scene-format.proto

```diff
@@ -0,0 +1,62 @@
+syntax = "proto3";
+
+package turbocow;
+
+message Scene {
+	enum MaterialType {
+		SOLID = 0;
+		GLASS = 1;
+		// other matherials
+	}
+	message Vector3 {
+		float x = 2;
+		float y = 3;
+		float z = 4;
+	}
+	message Transform {
+		int32 id = 5;
+		Vector3 translation = 6;
+		Vector3 rotation = 7;
+		Vector3 scale = 8;
```

Comment:
Btw, shouldn't this parameter be defined by 3  dimensions?

Reply:
I think yes